addons/project/views/project_views.xml

```diff
@@ -416,37 +416,6 @@
             </field>
         </record>
 
-        <record id="project_view_kanban" model="ir.ui.view">
```

Comment:
You can remove this view in a migration script. ;)

Reply:
I forget to add the PR in taks :) https://github.com/odoo/upgrade/pull/1632